src/tox_travis/hooks.py

```diff
@@ -17,5 +20,17 @@ def tox_addoption(parser):
 @tox.hookimpl
 def tox_configure(config):
     """Check for the presence of the added options."""
+
+    if 'TRAVIS' not in os.environ:
+        return
```

Comment:
There is a similar check in `toxenv.default_toxenv()`. @ryanhiebert - This seems like the correct place to perform the `TRAVIS` env check. Should the duplicate check in `toxenv.default_toxent()` be removed?